packages/ui/components/select-rich/src/LionSelectRich.js

```diff
@@ -369,6 +369,8 @@ export class LionSelectRich extends SlotMixin(ScopedElementsMixin(OverlayMixin(L
     if (this.hasNoDefaultSelected) {
       this._noDefaultSelectedInheritsWidth();
     }
+
+    this._listboxNode.setAttribute('autofocus', '');
```

Comment:
```suggestion
    // When `trapsKeyboardFocus:true` is configured in our overlay, we need to make sure
    // that our element with [role=listbox] gets focus. The `containFocus` util will look 
    // for an element with [autofocus] (otherwise it sets focus to `contentNode` (the 'root' of the overlay))
    this._listboxNode.setAttribute('autofocus', '');
```